recipes/django-localflavor/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "django-localflavor" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24e9a03bc11ced92365821d3cfdd96eb4e2f3e44a6f40e86c645d3e0ec65ec53
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
I don't think this package contains any C code so I think the `compiler` (and the whole `build` section) can go.

Reply:
build section removed, thanks!